clientloop.c

```diff
@@ -357,7 +357,7 @@ client_x11_get_proto(struct ssh *ssh, const char *display,
 					/* Don't overflow on long timeouts */
 					x11_timeout_real = UINT_MAX;
 				}
-				xasprintf(&cmd, "%s -f %s generate %s %s "
+				xasprintf(&cmd, "\"%s\" -f %s generate %s %s "
```

Comment:
Please have this only for windows although it doesn't make any regression to unix code.

I would like to reproduce the problem myself so please provide the steps.

Reply:
@bagajjal Done.